logic/transaction.js

```diff
@@ -1179,6 +1183,12 @@ class Transaction {
 				.join(', ')}`;
 		}
 
+		// Sanitize signatures property
+		transaction.signatures =
```

Comment:
@4miners Good Catch! Changed for setting it to null for having the property always present but not altering the checks for signatures type being array (this is also what is done on save so it's coherent in that sense)

Reply:
Ended up removing this change as it will not validate against the schemas I think the issue of the signatures property needs a deeper discussion. @4miners @diego-G